### PR TITLE
204 create model for student review requests

### PR DESCRIPTION
**Pull request by aedmondson**
I ended up making it into two models, let me know what you think of the schema.

### CODE REVIEW

**Review by zambroseo — APPROVED** [review]
found no issues, will also do a code review on it later tonight

File: ComSemApp/models.py [hunk 1]
```diff
@@ -334,11 +334,54 @@ class SpeakingPracticeAttempt(models.Model):
     wpm = models.DecimalField(max_digits=5,decimal_places=2, validators=[MinValueValidator(0)], verbose_name='Words per Minute')
 
     def __str__(self):
-        return f"{self.id}: {self.expression} - {self.correct}% ({self.date.strftime('%d %b %Y')})"
+        return f"{self.pk}: {self.expression} - {self.correct}% ({self.date.strftime('%d %b %Y')})"
```
Comment (dpmacisaac) [inline ComSemApp/models.py:337]: why was this changed? jus curious

File: ComSemApp/models.py [hunk 2]
```diff
@@ -334,11 +334,54 @@ class SpeakingPracticeAttempt(models.Model):
...
+    # The SpeakingPracticeAttemptReviewRequest that this review is satisfying
+    request = models.OneToOneField(SpeakingPracticeAttemptReviewRequest, on_delete=models.CASCADE, primary_key=True)
+    # The Teacher that reviewed the attempt
+    reviewer = models.ForeignKey(Teacher, on_delete=models.SET_NULL, null=True)
+    # The date on which the review was completed
+    date = models.DateField(auto_now_add=True)
+    # The comments the teacher has provided during review
+    comments = models.TextField(null=True, blank=True, verbose_name="Teacher Comments")
+    # The original score of the attempt. If blank, the score was not updated
+    original_score = models.DecimalField(null=True, blank=False, max_digits=5, decimal_places=2, validators=[MinValueValidator(0), MaxValueValidator(100)], verbose_name='Original Score')
```
Comment (dpmacisaac) [inline ComSemApp/models.py:380]: I am assuming that we plan on storing the teacher score by overwriting the score in the attempt?

**Review by dpmacisaac — APPROVED** [review]
looks good, lmk about my comments and then I'll go ahead and merge it in

File: ComSemApp/models.py (hunk 1 above)
Comment (aedmondson) [reply ComSemApp/models.py:337]: While `id` _technically_ works, the proper way to do it django is with `pk`. This way brings in type hinting and some nice syntax highlighting.

File: ComSemApp/models.py (hunk 2 above)
Comment (aedmondson) [reply ComSemApp/models.py:380]: Correct. I'll update the comment to reflect the change.
